x-pack/plugins/ml/public/application/jobs/new_job/pages/components/summary_step/summary.tsx

```diff
@@ -131,6 +135,13 @@ export const SummaryStep: FC<StepProps> = ({ setCurrentStep, isCurrentStep }) =>
           <EuiSpacer size="m" />
           <JobDetails />
 
+          {isAdvanced === false && (
+            <StartDatafeedSwitch
```

Comment:
Maybe worth disabling the switch like we do for the Create button once the job is running?

![image](https://user-images.githubusercontent.com/7405507/93327143-cfa43880-f811-11ea-8942-1ca0849aca85.png)

Reply:
good point.
fixed in https://github.com/elastic/kibana/pull/77484/commits/2ec87e11e9621f7f63f4c369de787e8995ffdd9a